lib/request.coffee

```diff
@@ -205,7 +244,57 @@ module.exports = getRequest = ({ token, debug = false, retries = 0, isBrowser =
 				responseError = data or 'The request was unsuccessful'
 				debugRequest(options, download.response)
 				throw new errors.ResinRequestError(responseError, download.response.statusCode)
+		.then(interceptResponse, interceptResponseError)
+
+	###*
+	# @summary Array of interceptors
+	# @type {Interceptor[]}
+	# @public
+	#
+	# @description
+	# The current array of interceptors to use. Interceptors intercept requests made
+	# by calls to `.stream()` and `.send()` (some of which are made internally) and
+	# are executed in the order they appear in this array for requests, and in the
+	# reverse order for responses.
+	#
+	# @example
+	# resin.interceptors.push(
```

Comment:
Good spot. I've gone for `request` in the end, for consistency with the other examples.